sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -3035,7 +3035,10 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
     val msg = intercept[AnalysisException] {
       sql("REFRESH FUNCTION md5")
     }.getMessage
-    assert(msg.contains("Cannot refresh builtin function"))
+    assert(msg.contains("Cannot refresh built-in function"))
+    intercept[NoSuchFunctionException] {
```

Comment:
Just in case, could you check the error message, too?

Reply:
added.